components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/ClaimUtil.java

```diff
@@ -384,6 +384,22 @@ public static boolean isMultiValuedAttribute(String claimValue) {
         return StringUtils.contains(claimValue, ATTRIBUTE_SEPARATOR);
     }
 
+    /**
+     * Check whether claim value is multivalued attribute or not by using attribute separator.
```

Comment:
Is this description valid for the new method?

Reply:
need to change to `Check whether claim value is multivalued attribute or not by using claim uri and attribute separator.`